docs/standard/exceptions/creating-localized-exception-messages.md

```diff
@@ -0,0 +1,113 @@
+---
+title: How to: create custom exceptions and localized exception messages
+description: The article demonstrates how to create custom exceptions and localized exception messages
+author: Youssef1313
+ms.date: 09/02/2019
+---
+# How to: create custom exceptions and localized exception messages
+
+Written by [Youssef Victor](https://github.com/Youssef1313).
+
+## Create custom exceptions
+The .NET Framework contains many different exceptions that are likely to meet your needs. However, in some cases when none of them meets your needs, you can create your own custom exceptions.
+Let's assume you want to create a `StudentNotFoundException` that contains a `StudentName` property.
+To create custom exceptions, Follow these steps:
+
+- Create a serializable class that inherits from `Exception`. The class name is preferred to end in "Exception".
+
+```csharp
```

Comment:
The code here and throughout needs to be indented so that it aligns with the bullets. Here is what it looks like now:

![image](https://user-images.githubusercontent.com/10886961/64463322-79f26780-d0b8-11e9-96ef-7728aab57e02.png)

Reply:
I'm not sure if it will render correctly now ?